lib/Controller/Playlist.php

```diff
@@ -141,6 +141,7 @@ public function displayPage(Request $request, Response $response)
         // Call to render the template
         $this->getState()->template = 'playlist-page';
         $this->getState()->setData([
+            'users' => $this->userFactory->query(),
```

Comment:
This approach doesn't scale on CMS instances with 100s of users - we need to use select2 instead.

Reply:
I'll do that for this case then and also the Layout Editor page where we use the same approach.